core/services/relay/evm/evm.go

```diff
@@ -390,6 +394,8 @@ func newContractTransmitter(lggr logger.Logger, rargs commontypes.RelayArgs, tra
 		return nil, pkgerrors.New("no sending keys provided")
 	}
 
+	configWatcher := opts.configWatcher
```

Comment:
nit. this seems like a required option (code below will fail if this is nil) so not sure if it made more sense as a parameter before (unlike pluginGasLimit for example)